### DISCUSSION TITLE
[Feature] Search people on mobile

### DISCUSSION THREAD

**Discussion by bigstusexy** [immich-app/immich#14734] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

On mobile, getting to a person is a very daunting and long process if many names.  If you go to search, there is a selector for people that brings up a list.  One cannot search this list, they must scroll too it.   If you go to Library, People, there is a screen full of people, again this is not searchable.   

It would be great if there was a  search box on these lists, possibly one that maybe even instantly searched like the way the filter people in search does on the web. 

I think I've been using this since 1.112, however the app version I'm using now is 1.22.3 for Android, from the Play store. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#11749
